swagger.json

```diff
@@ -1121,6 +1121,77 @@
             }
           }
         },
+        "/fleet/drivers/{driver_id}/hos_daily_log": {
+            "patch": {
+              "consumes": [
+                "application/json"
+              ],
+              "description": "Modify details, such as trailer number or shipping ID, for a driver's daily log",
```

Comment:
probably a PM question, do we want to make it explicit here that we are going to append this new information rather than overwriting it?

Reply:
If I wanted to faithfully model how the UI does it, it would seem to make the most sense to have it overwritten since the data model is just one large string instead of a list. Perhaps this can be clearer in the description